shared/reducers/config.tsx

```diff
@@ -25,7 +25,9 @@ export default function(state: Types.State = initialState, action: Actions): Typ
       return state.merge({
         configuredAccounts: state.configuredAccounts,
         defaultUsername: action.payload.wasCurrentDevice // if revoking self find another name if it exists
-          ? state.configuredAccounts.find(n => n !== state.defaultUsername) || ''
+          ? I.List(state.configuredAccounts.map(account => account.username)).find(
```

Comment:
you should find then map instead of mapping all the stuff you're going to throw away

Reply:
I'm only throwing away one row - I find it implausible that all the extra indirection from checking `.username` on all the rows I'm mapping is worth not mapping that one row.

Reply:
I'm confused why it's one row. Please explain

Reply:
I misread the code as `filter` not `find`.. you're right, fixing.